extensions/resteasy-reactive/rest-client-reactive/deployment/src/test/java/io/quarkus/rest/client/reactive/converter/ParamConverterProviderTest.java

```diff
@@ -117,7 +117,7 @@ interface Client {
 
         @GET
         @Path("/cookie")
-        String getWithCookie(@HeaderParam("cookie-param") Param param);
+        String getWithCookie(@CookieParam("cookie-param") Param param);
```

Comment:
This change is unrelated to this pull request, but I saw it was originally wrong, so I fix it to use the `@CookieParam` annotation.